main.go

```diff
@@ -537,9 +560,9 @@ func (cacheModel *CacheModel) CleanPaths() error {
 		}
 	}
 	cacheModel.PathList = cleanedPathList
-
 	cleanedIgnoredPathList := []string{}
-	for _, path := range cacheModel.IgnoreList {
+	ignoreListWithoutDuplicates := cleanDuplicatePaths(cacheModel.IgnoreList)
```

Comment:
Please do not filter ignore patterns, it is not required as it is not an issue if one pattern is checked multiple times for a path.

Reply:
done